kernel/src/fs/procfs/pid/mod.rs

```diff
@@ -16,6 +16,8 @@ mod cmdline;
 mod comm;
```

Comment:
This PR overall looks fine to me. I wonder if the gvisor has tests on "/proc/[pid]/stat" or "/proc/[pid]/status" that we can ensure that we provide the correct fields and the behavior?

Reply:
Yes. I add the fields which we have already defined and used in other code blocks.